packages/datadog-plugin-pg/src/index.js

```diff
@@ -32,9 +32,30 @@ function createWrapQuery (tracer, config) {
       }
 
       const originalCallback = pgQuery.callback
-      const statement = pgQuery.text
+      const statement = (pgQuery.cursor && pgQuery.cursor.text) || pgQuery.text
       const params = this.connectionParameters
 
+      function finishSpan (err) {
+        if (err) {
```

Comment:
This block can be replaced with `span.setTag('error', err)`. Most of the plugins using the `if` and the `addTags` syntax were built before the functionality was added to the tagger.